src/layouts/app/connection-mixin.js

```diff
@@ -80,7 +80,7 @@ export default superClass =>
             throw err;
           }
         },
-        callApi: async (method, path, parameters) => {
+        callApi: async (method, path, parameters, responseType = '') => {
```

Comment:
I don't think we need default here. Just pass undefined and let hassCallApi handle default